run_operations.py

```diff
@@ -16,10 +16,10 @@
 ############## DIRECTORY AND FILE CONSTANTS ##############
 OPERATIONS_DIRECTORY = os.getenv('OPERATIONS')
 SCRATCH_DIRECTORY = os.getenv('SCRATCHDIR')
-TEMPLATE_DIRECTORY = OPERATIONS_DIRECTORY + "/TEMPLATES"
-LOGS_DIRECTORY = OPERATIONS_DIRECTORY + "/LOGS"
-ERRORS_DIRECTORY = OPERATIONS_DIRECTORY + "/ERRORS"
-STORED_DATE_FILE = OPERATIONS_DIRECTORY + "/stored_date.date"
+TEMPLATE_DIRECTORY = os.path.join(OPERATIONS_DIRECTORY, "/TEMPLATES")
```

Comment:
I think the `/` needs to be removed - won't this discard `OPERATIONS_DIRECTORY` since `/TEMPLATES` is an absolute path?

Reply:
Good catch